mycore-base/src/main/java/org/mycore/frontend/cli/MCRCommandLineInterface.java

```diff
@@ -321,19 +325,35 @@ public static void getURI(String uri, String file) throws Exception {
     public static List<String> readCommandsFile(String file) throws IOException {
         try (BufferedReader reader = Files.newBufferedReader(new File(file).toPath(), Charset.defaultCharset())) {
             output("Reading commands from file " + file);
+            return readCommandsFromBufferedReader(reader);
+        }
+    }
+
+    public static List<String> readCommandsRessource(String resource) throws IOException {
+        final URL resourceURL = MCRClassTools.getClassLoader().getResource(resource);
+        if (resourceURL == null) {
+            return Collections.emptyList();
+        }
+        try (InputStream is = resourceURL.openStream()) {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(is, Charset.defaultCharset()))) {
```

Comment:
Better link Autoclosables in one line
`try (InputStream is=..; InputStreamReader ir=...; BufferedReader reader=...)`
Consider NOT using "`defaultCharset()`" for resources but use a defined one like `StandardCharsets.UTF_8`